wordpress/resources/install.yaml

```diff
@@ -0,0 +1,14 @@
+ apiVersion: batch/v1
+ kind: Job
+ metadata:
+   name: install
+ spec:
+   template:
+     metadata:
+       name: install
+     spec:
+       restartPolicy: OnFailure
+       containers:
+         - name: install
+           image: quay.io/gravitational/debian-tall:0.0.1
+           command: ["/usr/local/bin/helm", "install", "/var/lib/gravity/resources/charts/wordpress", "--set", "registry=leader.telekube.local:5000/","--name","wordpress"]
```

Comment:
Nit: Could you make this consistent with how command looks in the upgrade hook please? I.e. specify list like this:
```yaml
command:
  - /usr/local/bin/helm
  - ...
```

Will be a bit easier to read too.